internal/transform/span.go

```diff
@@ -239,8 +238,6 @@ func (m *mapper) getHTTPInfo(attrs map[string]interface{}) *telemetry.SpanHttpIn
 
 	if reqBody, ok := attrs["http.request_body"]; ok {
 		spanHttpInfo.Request.Body = fmt.Sprint(reqBody)
-	} else {
-		m.logger.Error("unable to fetch HTTP request body")
```

Comment:
Why did you delete it?

Reply:
because not all http traffic has body like GET